### PR TITLE
Implement gRPC client in Electron.

### PR DESCRIPTION
**Pull request by anxolerd**
Implement simple client which basically works. The implementation can be improved, but not in this PR.

Currently supports only `UnaryCall <-> UnaryCall` calls

### CODE REVIEW

**Comment by anxolerd** [comment]
Just noticed a lot of code is missing here. :(
Marking WIP until issue is resolved.

**Comment by anxolerd** [comment]
Code is restored. Can be reviewed and merged

File: package.json [hunk 1]
```diff
@@ -8,7 +8,9 @@
     "react-dom": "^16.0.0",
     "react-redux": "^5.0.6",
     "react-scripts": "1.0.14",
-    "redux": "^3.7.2"
+    "redux": "^3.7.2",
+    "redux-logger": "^3.0.6",
```
Comment (vsel) [inline package.json]: Точно не в devDependency?

File: src/actions/types.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+export const URL = {
+  HOST_CHANGED: 'URL-HOST_CHANGED',
+  PORT_CHANGED: 'URL-PORT_CHANGED',
```
Comment (vsel) [inline src/actions/types.js]: - и _ это специально?

File: src/lib/utils.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+const compose = (...fns) => (...args) => (
```
Comment (vsel) [inline src/lib/utils.js]: Обьясниш потом почему именно так

File: package.json (hunk 1 above)
Comment (anxolerd) [reply package.json]: Hm.... I import this in the application code explicitly under no condition. Not sure whether I should and can move it to devDependencies right now. But it have to be there in the end.

File: src/lib/utils.js (hunk 3 above)
Comment (anxolerd) [reply src/lib/utils.js]: This is a simple function composition. Nothing really difficult. Mb state more concrete what is hard to understand, so I could give more precise answer.

**Pull request merged**